connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/ClientActorRefs.java

```diff
@@ -29,7 +30,7 @@
  * Collection of all client actor refs of a connection actor.
  */
 @NotThreadSafe
-public final class ClientActorRefs {
+public final class ClientActorRefs implements AkkaJacksonCborSerializable {
```

Comment:
We should add a unit test for serialization.
You could have a look at `JsonValueSourceRefTest` for an example. (PS: nowadays I would use `ActorSystemResource` as ClassRule.)